spec/beautify-languages-spec.coffee

```diff
@@ -44,12 +44,11 @@ describe "BeautifyLanguages", ->
 
   # Activate all of the languages
   allLanguages = [
-    "c", "clojure", "coffee-script", "css", "d", "html",
-    "java", "javascript", "json", "less",
-    "mustache", "objective-c", "perl", "php",
-    "python", "ruby", "sass", "sql", "svg",
-    "xml", "csharp", "gfm", "marko",
-    "go", "html-swig", "lua"
+    "c", "clojure", "coffee-script", "css", "csharp", "d", "fuse",
+    "gfm", "go", "html", "html-swig", "java", "javascript",
+    "json", "less", "lua", "marko", "mustache", "objective-c",
+    "perl", "php", "python", "ruby", "sass", "sql",
+    "svg", "xml"
```

Comment:
I am not sure why but adding language-fuse to the build artifacts makes the build time out.